HISTORY.md

```diff
@@ -1,3 +1,6 @@
+- Revert "Add destination mw (#148)"
```

Comment:
Let's remove the "Revert" line, since it will be i the release log, and confuses people

Reply:
resolved by https://github.com/segmentio/analytics.js-core/pull/150/commits/fccfece9c33838866a52cb792e85cc3f3060d51b